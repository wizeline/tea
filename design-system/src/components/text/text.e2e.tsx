import { newE2EPage } from '@stencil/core/testing';

describe('wz-text', () => {
  it('renders text component', async () => {
    const page = await newE2EPage();
    await page.setContent('<wz-text>Hello, my name is Marty McFly</wz-text>');
    const text = await page.find('wz-text');
    expect(text.innerHTML).toMatchSnapshot();
  });
});
