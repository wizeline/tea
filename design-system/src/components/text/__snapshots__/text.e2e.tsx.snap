// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-text renders text component 1`] = `"<span class=\\"regular default\\">Hello, my name is Marty McFly</span>"`;
